INSTALL-Mac.md

```diff
@@ -125,6 +125,14 @@ The generated executables will appear in `ELL/build/bin`.
 
 ## Troubleshooting
 
+**LLVM not found, please check that LLVM is installed.**
+
+Try telling CMake where to find LLVM as follows:
+
+```shell
+cmake -DLLVM_DIR=/usr/local/Cellar/llvm@3.9/3.9.1_1/lib/cmake/llvm ..
+```
+
 **ELL/libraries/model/include/IRModelProfiler.h:15:10: fatal error: 'EmitterTypes.h' file not found #include "EmitterTypes.h"**
```

Comment:
I think your new instructions supersede this blurb, no? Maybe they could be merged. "If setting `LLVM_DIR` fixes your issues, you may want to consider creating a symlink.".

Reply:
I'm not sure, because I didn't see this issue...

Reply:
In any case, that troubleshooting instruction didn't work for me in a clean setup, so I removed it.  We can always add it back later.